Add styles for selected tab

bpaduch opened the pull request:
- Added CSS to make tabs appear like tabs
- Fixes https://github.com/ampproject/docs/issues/510.
- Used @morsssss's [code](https://github.com/morsssss/amp-by-example/commit/d6c235b25c50a168cc547d0332feec1ce1baac5f) with a couple tweaks.
- Tested on embedded samples with default active tab of "Source" and specified active tab "Preview'

![tabs-new-look](https://user-images.githubusercontent.com/1012254/27838116-7934d9aa-60b6-11e7-8e02-2a392661873f.jpg)

to: @sebastianbenz, @pbakaus

sebastianbenz commented:
LGTM

I think it looks a bit weird that the tabs have a border, but not the tab panels. How about something like this?

<img width="454" alt="screen shot 2017-07-04 at 7 42 56 pm" src="https://user-images.githubusercontent.com/380472/27840472-2633f5ac-60f1-11e7-8c64-901314fd0cbb.png">

<img width="462" alt="screen shot 2017-07-04 at 7 42 50 pm" src="https://user-images.githubusercontent.com/380472/27840475-29c06cbe-60f1-11e7-96b5-42984a9202b6.png">

bpaduch commented:
Good suggestions; CSS is not my forte but after much trial/error, I think I got it.
- Added border around source panel (had to remove width=100%) for border to appear on right.
- Only need top border for preview panel; looks weird with border all around (especially for content that doesn't fit full panel)
- Removed code block box-shadow - doesn't work with borders and not appealing for tabs.

![tabs-new-look2](https://user-images.githubusercontent.com/1012254/27844491-8fa2c4ce-60f0-11e7-98e7-c260ab2f946b.jpg)

sebastianbenz commented:
Looks great! One last thing I've just noticed. Maybe align the tabs with the panels by removing the padding-left for the ul.

bpaduch commented:
Thanks! Hmm, I kind of liked that padding and kept it. :)

sebastianbenz commented:
😀 - no problem

sebastianbenz commented:
published to NPM
